### ISSUE TITLE
Marker distorted into a square

### ISSUE THREAD

**Issue by AnaSasso** [goat-community/goat#3094]

**Comment by majkshkurti** [comment]
I think this is because the icon should have a 1:1 ratio (width=height). With other icons this doesn't happen.

**Comment by AnaSasso** [comment]
> I think this is because the icon should have a 1:1 ratio (width=height). With other icons this doesn't happen.

does it help if I check which ones this happen and provide you with an .svg with the right ration?

**Comment by majkshkurti** [comment]
Yes, that would help. Here is a reference of a maki icon that works: 

https://assets.plan4better.de/icons/maki/airport.svg

It would be also good to have the svgs at 15x15 px.
